src/meta-srv/src/procedure/region_migration/downgrade_leader_region.rs

```diff
@@ -331,41 +316,6 @@ mod tests {
         assert!(!err.is_retryable());
     }
 
-    #[tokio::test]
-    async fn test_should_reject_writes() {
```

Comment:
Do we still need to write a unit test for rejecting writes?

Reply:
Now, rejecting writes has become the default behavior, we don't need this unit test anymore